issue: Cannot create a new project on ppdd-manager

# Issue report
**Logs**
java.util.NoSuchElementException: None.get
        at scala.None$.get(Option.scala:529)
        at scala.None$.get(Option.scala:527)
        at ppddm.manager.controller.project.ProjectController$.$anonfun$createProject$2(ProjectController.scala:34)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
15/09/2020 06:40:50 ERROR ppddm.manager.gateway.ManagerHttpServer$: DBException: Error while inserting a Project with project_id:9a1a9d4a-a3f9-4a7b-9b5c-517803d9bb99 into the database.
ppddm.core.exception.DBException: Error while inserting a Project with project_id:9a1a9d4a-a3f9-4a7b-9b5c-517803d9bb99 into the database.
        at ppddm.manager.controller.project.ProjectController$$anonfun$createProject$3.applyOrElse(ProjectController.scala:41)
        at ppddm.manager.controller.project.ProjectController$$anonfun$createProject$3.applyOrElse(ProjectController.scala:37)
        at scala.runtime.AbstractPartialFunction.apply(AbstractPartialFunction.scala:38)
        at scala.util.Failure.recover(Try.scala:234)
        at scala.concurrent.Future.$anonfun$recover$1(Future.scala:395)
        at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
        at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
        at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
        at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
        at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
        at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
        at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
        at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
        at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
Caused by: java.util.NoSuchElementException: None.get
        at scala.None$.get(Option.scala:529)
        at scala.None$.get(Option.scala:527)
        at ppddm.manager.controller.project.ProjectController$.$anonfun$createProject$2(ProjectController.scala:34)
        at scala.util.Success.$anonfun$map$1(Try.scala:255)
        at scala.util.Success.map(Try.scala:213)
        at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
        ... 9 more

--- a/ppddm-manager/src/main/scala/ppddm/manager/controller/project/ProjectController.scala
+++ b/ppddm-manager/src/main/scala/ppddm/manager/controller/project/ProjectController.scala
@@ -31,7 +31,7 @@ object ProjectController {
     db.getCollection[Project](COLLECTION_NAME).insertOne(projectWithId).toFuture() // insert into the database
       .map { result =>
         val _id = result.getInsertedId.asObjectId().getValue.toString
-        logger.debug("Inserted document _id:{} and projectId:{}", _id, project.project_id.get)
+        logger.debug("Inserted document _id:{} and projectId:{}", _id, projectWithId.project_id.get)
         projectWithId
       }
       .recover {

